docs/static/arcsight-module.asciidoc

```diff
@@ -17,6 +17,16 @@ With a single command, the module taps directly into the ArcSight Smart Connecto
 parses and indexes the security events into Elasticsearch, and installs a suite of Kibana dashboards 
 to get you exploring your data immediately. 
 
+[[arcsight-prereqs]]
+==== Prerequisites
+
+These instructions assume that Logstash, Elasticsearch, and Kibana are already
+installed. The products you need are https://www.elastic.co/downloads[available
+to download] and easy to install. The Elastic Stack 5.6 or later is required for
```

Comment:
It's probably a good idea to note that x-pack is required in the installation. i.e. "The Elastic Stack 5.6 or later and X-Pack are required for this module."